readme.md

```diff
@@ -27,6 +27,7 @@ Force reload the window.
 
 Right-click on an HTML element when DevTools is open to inspect it.
 
+Note: this features has been moved to [its own package](https://github.com/parro-it/debug-menu). Please refer to it for related issues.
```

Comment:
Can you just remove this "Inspect element" section and add your module at the bottom in a `Related` section. Like, eg https://github.com/sindresorhus/got#related